pySDC/implementations/convergence_controller_classes/adaptive_collocation.py

```diff
@@ -16,11 +16,11 @@ class AdaptiveCollocation(ConvergenceController):
 
     Use this convergence controller by supplying parameters that the sweeper accepts as a list to the `params`.
     For instance, supplying
-    ```
-    params = {
-        'num_nodes': [2, 3],
-    }
-    ```
+
+        params = {
+            'num_nodes': [2, 3],
+        }
+
```

Comment:
The ticks indicate that this is code. Why did you remove them?

Reply:
Because they were not rendered correctly (check [the website](https://parallel-in-time.org/pySDC/pySDC/implementations.convergence_controller_classes.adaptive_collocation.html#implementations.convergence_controller_classes.adaptive_collocation.AdaptiveCollocation) ). In local deployment it got rendered correctly with this new syntax

Reply:
@lisawim, what do you usually do in this case? I believe you have quite some experience with writing documentation :D

Reply:
Actually, code blocks in docstring should probably written like this : https://stackoverflow.com/questions/64451966/python-sphinx-how-to-embed-code-into-a-docstring

Or a more "python-like" approach could be used : https://gitlab.com/tlunet/casper/-/blob/master/casper/maths/analytic.py?ref_type=heads#L79 (not sure how it is rendered by sphinx, tho)

Reply:
What @tlunet said. :D

Reply:
Thank you @tlunet for the links. I now included the first way in the file. I didn't know about those and "simply" indenting them already makes it keep formatting, but marking it as code is way better.